man/systemd-analyze.xml

```diff
@@ -792,6 +792,315 @@ Service b@0.service not loaded, b.socket cannot be started.
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_NETLINK</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_AF_PACKET</entry>
+              </row>
+              <row>
+                <entry>RestrictAddressFamilies_OTHER</entry>
+              </row>
+              <row>
+                <entry>SystemCallArchitectures</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_swap</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_obsolete</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_clock</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_cpu_emulation</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_debug</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_mount</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_module</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_raw_io</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_reboot</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_privileged</entry>
+              </row>
+              <row>
+                <entry>SystemCallFilter_resources</entry>
+              </row>
+              <row>
+                <entry>IPAddressDeny</entry>
+              </row>
+              <row>
+                <entry>DeviceAllow</entry>
+              </row>
+              <row>
+                <entry>AmbientCapabilities</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
+        </listitem>
+        </varlistentry>
+
+      <example>
+        <title>JSON Policy</title>
+        <para>The JSON file passed as a path parameter to <option>--security-policy=</option>
+        has a top-level JSON object, with keys being the assessment test identifiers mentioned
+        above. The values in the file should be JSON objects with one or more of the
+        following fields: description_na (string), description_good (string), description_bad
+        (string), weight (unsigned integer), and range (unsigned integer). If any of these fields
+        corresponding to a specific id of the unit file is missing from the JSON object, the
+        default built-in field value corresponding to that same id is used for security analysis
+        as default. The weight and range fields are used in determining the overall exposure level
+        of the unit files so by allowing users to manipulate these fields, 'security' gives them
+        the option to decide for themself which ids are more important and hence, should have a greater
+        effect on the exposure level. </para>
+
+{
```

Comment:
enclose the JSON data in `<programlisting>`.

Please build and look at the man page before submitting the PR